src/v0/destinations/braze/util.js

```diff
@@ -257,14 +259,16 @@ const BrazeDedupUtility = {
     const keys = Object.keys(userData)
       .filter((key) => !excludeKeys.includes(key))
       .filter((key) => !BRAZE_NON_BILLABLE_ATTRIBUTES.includes(key))
-      .filter(
-        (key) =>
-          !(
+      .filter((key) => {
+        if (isObject(userData[key])) {
+          return !(
             Object.keys(userData[key]).includes('$add') ||
```

Comment:
You can use `in` operator to reduce the code

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in